pkg/upstream/cluster/host.go

```diff
@@ -57,6 +58,7 @@ func NewSimpleHost(config v2.Host, clusterInfo types.ClusterInfo) types.Host {
 		tlsDisable:    config.TLSDisable,
 		weight:        config.Weight,
 		healthFlags:   GetHealthFlagPointer(config.Address),
+		startTime:     time.Now(),
```

Comment:
Initialize only when slow start is enabled.

Reply:
I will deal with this later.